packages/react/tsconfig.json

```diff
@@ -3,7 +3,15 @@
   "compilerOptions": {
     "outDir": "dist/esm",
     "rootDir": "src",
-    "tsBuildInfoFile": "tsconfig.esm.tsbuildinfo"
+    "tsBuildInfoFile": "tsconfig.esm.tsbuildinfo",
+    "plugins": [
+      {
+        "transform": "ts-transform-define",
+        "replace": {
+          "isCacheDisabled()": "false"
+        }
+      }
+    ]
```

Comment:
This will only apply to the `esm` build - the other builds for React use `"extends": "../tsconfig.options.json",` as well, so they won't pick up changes.

Check with @MonicaOlejniczak what might be the best way forward, as I believe she was working on these files recently.

Reply:
I ended up creating a common config so we don't duplicate code here